packages/shared/src/components/modals/OnboardingModal.tsx

```diff
@@ -177,7 +162,7 @@ function OnboardingModal({
     return (
       <>
         <IntroductionOnboarding onClose={onClose} />
-        {onboardingSteps.map((onboarding) => components[onboarding])}
+        <FilterOnboardingStep key={OnboardingStep.Topics} />
```

Comment:
I think we can let go of the key now.
```suggestion
        <FilterOnboardingStep />
```